cmd/timoni/mod_import_crd_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2023 Stefan Prodan
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+
+	"github.com/mattn/go-shellwords"
+	. "github.com/onsi/gomega"
+)
+
+func TestImportCrd(t *testing.T) {
```

Comment:
this looks like it effectively covers the same functionality as the modified tests from `internal/engine`, right? and with much cleaner golden files than that crappy string

Reply:
Oh yeah, this gives you a readable output which is easy to understand since it uses `git diff`. You see spot the differences and if needed you can also regenerate the golden files with a locally built binary.